autoPyTorch/pipeline/components/training/trainer/base_trainer_choice.py

```diff
@@ -196,34 +194,13 @@ def fit(self, X: Dict[str, Any], y: Any = None, **kwargs: Any) -> autoPyTorchCom
                    ] if 'logger_port' in X else logging.handlers.DEFAULT_TCP_LOGGING_PORT,
         )
 
-        fit_function = self._fit
-        if X['use_pynisher']:
-            wall_time_in_s = X['runtime'] if 'runtime' in X else None
-            memory_limit = X['cpu_memory_limit'] if 'cpu_memory_limit' in X else None
-            fit_function = pynisher.enforce_limits(
-                wall_time_in_s=wall_time_in_s,
-                mem_in_mb=memory_limit,
-                logger=self.logger
-            )(self._fit)
-
         # Call the actual fit function.
-        state_dict = fit_function(
+        self._fit(
```

Comment:
Do we need to return the state_dict on line [339](https://github.com/franchuterivera/Auto-PyTorch/blob/de4ae71bcc3891f1bd9c238b0b876dfd6deb8ed1/autoPyTorch/pipeline/components/training/trainer/base_trainer_choice.py#L339)